web/server.js

```diff
@@ -7,8 +7,16 @@ const app = next({dev}); //Create the next.js server
 const handleNextRequest = app.getRequestHandler(); //We need to get the next.js request handler so any routes we don't handle manually
                                                     //will be handled by Next.JS
 const {pool} = require('./utils/query')
-const passport = require('passport')
 const auth = require('./middleware/auth')
+const passport = require('passport')
+
+passport.serializeUser(function(user, done) {
+  done(null, user)
+})
+
+passport.deserializeUser(function(user, done) {
+  done(null, user)
```

Comment:
So what this function does it say to passport: "Hey, I have a user looking to get their session loaded. Their user id is ____, do they exist? If they do, stick their profile into memory for me."

When you use the function(user, done) you're assuming that the user is looking themselves up using their own object, which they don't have.

The proposed solution for this is as follows
```js
passport.deserializeUser(function(id, done) {
//Psueodocode for database call here
//Look up users table for the specified ID.
let user = await SOME_USER_LOOKUP_FUNCTION
if(user) {
  done(null, user)
} else {
  done(null, false)
}
})
```

Reply:
Oh okay makes a lot of sense now, will work on the lookupById function and update this